weather_monitoring_operations/test_views.py

```diff
@@ -0,0 +1,72 @@
+from rest_framework.test import APITestCase
+
+from django.urls import reverse
+from conftest import get_oauth2_token
+from unittest.mock import patch
+import unittest
+from .views import _fetch_weather_data
+from .weather_service import WeatherService
+import time
+
+
+JWT = get_oauth2_token()
+
+
+# Create your tests here.
+class WeatherMonitoringOperationsTestCase(APITestCase):
+    def setUp(self):
+        self.client.defaults["HTTP_AUTHORIZATION"] = "Bearer " + JWT
+        self.api_url = reverse("get_weather_data")
+
+    def test_get_weather(self):
+        response = self.client.get(self.api_url, content_type="application/json")
+
+        print(response.content)
+
+        assert response.status_code == 200
+
+        assert response["Content-Type"] == "application/json"
```

Comment:
```suggestion
        self.assert.Equal(response.headers.get("Content-Type","application/json")
```
We could use the `header` dictionary of the response object in this case :)